zephyr/esp32c3/CMakeLists.txt

```diff
@@ -60,6 +67,10 @@ if(CONFIG_SOC_ESP32C3)
 
   zephyr_sources(
     ../../components/soc/esp32c3/gpio_periph.c
+    ../../components/esp_timer/src/ets_timer_legacy.c
+    ../../components/esp32c3/clk.c
```

Comment:
This file has been updated. Can you rebase with latest version and test if it still works?